modules/shared/src/main/scala/io/constellationnetwork/currency/dataApplication/package.scala

```diff
@@ -543,4 +544,5 @@ trait L0NodeContext[F[_]] {
   def getCurrencySnapshot(ordinal: SnapshotOrdinal): F[Option[Hashed[CurrencyIncrementalSnapshot]]]
   def getLastCurrencySnapshotCombined: F[Option[(Hashed[CurrencyIncrementalSnapshot], CurrencySnapshotInfo)]]
   def securityProvider: SecurityProvider[F]
+  def getCurrencyId: F[Address]
```

Comment:
should this return an `Address` or a `CurrencyId`? It seems confusingly named if it is for the former

Reply:
Ooh you're right, I forgot we have the currencyId type now, let me update